pyanaconda/modules/payloads/source/cdrom/initialization.py

```diff
@@ -40,15 +40,23 @@ def _do_mount(self):
         """Run CD-ROM installation source setup."""
         log.debug("Trying to detect CD-ROM automatically")
 
+        device_name = self._search_for_optical_media()
```

Comment:
I would merge both commits that modify this task. The first commit is not really useful.